STATUS.md

```diff
@@ -0,0 +1,314 @@
+# DCS Functional Parity
+
+The DCS gRPC Server API implementation is still a work in progress, and therefore the following API functions have
+been implemented.
+
+The unimplemented functions is not committed to a roadmap. We do envision full DCS API equivalence at some point in the
+future; pull requests are welcomed for expanding the API equivalency.
+
+---
+
+## Singletons
+
+### Atmosphere Singleton
+- [x] `getWind`
+- [x] `getWindWithTurbulence`
+- [x] `getTemperatureAndPressure`
+
+### Coalitions Singleton
+- [ ] `addGroup`
+  - [ ] Sea
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+  - [ ] Ground
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+  - [ ] Plane
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+    - [ ] Loadout
+  - [ ] Helicopters
+    - [ ] Group
+    - [ ] Units
+    - [ ] Waypoints
+    - [ ] Tasks
+    - [ ] Loadout
+- [ ] `addStaticObject`
+- [x] `getGroups`
+- [ ] `getStaticObjects`
+- [x] `getAirbases`
+- [ ] `getPlayers`
+- [ ] `getServiceProviders`
+- [ ] `addRefPoint`
+- [ ] `getRefPoints`
+- [x] `getMainRefPoint` ("Bullseye")
+- [ ] `getCountryCoalition`
+
+### Coord Singleton
```

Comment:
I don't see a need for these in the public API. The cients should only be sending LL and we do the conversions internally.